### PR TITLE
Youtube integration

### PR DESCRIPTION
**Pull request by xannyxs**
Made the following changes to the project:
- Added an integration with YouTube on the back-end
~~An upload page, which does not work yet, but will follow up on the next PR~~
~~An user page, which is not complete yet.~~
- All cookies are now `secure`

Nothing has been done for the YouTube Integration on the front-end
Please add the following env in `new-server`: OAUTH_SECRET

### CODE REVIEW

File: packages/app/app/studio/user/page.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const User = async () => {
+  const isAuthorized = CheckAuthorization()
+
+  if (!isAuthorized) {
+    return <AuthorizationMessage />
+  }
+  const userData: IExtendedUser = await fetchUserAction({})
+
+  // *************************** //
```
Comment (xannyxs) [inline packages/app/app/studio/user/page.tsx:25]: Is hardcoded, and will be updated in the next PR

**Comment by xannyxs** [comment]
Removed front-end components except UI, which will be added in the next PR

File: packages/server/src/controllers/session.controller.ts
Comment (Eric-Vondee) [inline packages/server/src/controllers/session.controller.ts:1]: @xannyxs This should be a POST request

**Review by pabloVoorvaart — APPROVED** [review]
LGTM

**Pull request merged**